docs/feature_oled_driver.md

```diff
@@ -166,6 +166,40 @@ bool oled_task_user(void) {
 #endif
 ```
 
+Render a message before booting into bootloader mode.
+```c
+void oled_render_boot(void) {
+    oled_clear();
+    for (int i = 0; i < 16; i++) {
+        oled_set_cursor(0, i);
+        oled_write_P(PSTR("BOOT "), false);
```

Comment:
```suggestion
        if (bootloader) {
            oled_write_P(PSTR("Awaiting New Firmware "), false);
        } else {
            oled_write_P(PSTR("Rebooting "), false);
        }
```

Reply:
Not a bad change ... I'll probably roll that into my firmware down the road. I was using the boot lines to figure out why I couldn't get a complete image before figuring out the driver batched updates.